pkg/instancetype/instancetype_test.go

```diff
@@ -161,6 +163,9 @@ var _ = Describe("Instancetype and Preferences", func() {
 						CPU: instancetypev1alpha2.CPUInstancetype{
 							Guest: uint32(2),
 						},
+						Memory: instancetypev1alpha2.MemoryInstancetype{
```

Comment:
Is this change needed?

Reply:
Yeah the `Equals` on L321 wasn't happy without this required field also being set.

Reply:
ACK